apps/client/src/register/components/registerForm.tsx

```diff
@@ -0,0 +1,143 @@
+import { Button } from '@mui/material';
+import FormInput from '../../components/FormInput';
+import React, { useState } from 'react';
+import axios from 'axios';
+
+const RegisterForm: React.FC = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [confirmPassword, setConfirmPassword] = useState('');
+  const [firstName, setFirstName] = useState('');
+  const [lastName, setLastName] = useState('');
+  const [passwordError, setPasswordError] = useState('');
+  const [confirmPasswordError, setConfirmPasswordError] = useState('');
```

Comment:
Now that you know how to validate forms using `useState` hook, the next form will use `useReducer` keep this in mind.